src/mmw/js/src/modeling/views.js

```diff
@@ -494,19 +502,37 @@ var ModelingResultsWindow = Marionette.LayoutView.extend({
     animateIn: function() {
         var self = this;
 
-        this.$el.animate({ height: '55%' }, 400, function() {
+        this.$el.animate({ height: '55%', 'min-height': '300px' }, 200, function() {
             self.trigger('animateIn');
             App.map.set('halfSize', true);
+            $(self.ui.toggle.selector).blur()
+                .find('i')
+                    .removeClass('fa-angle-up')
+                    .addClass('fa-angle-down');
         });
     },
 
     animateOut: function() {
         var self = this;
 
-        this.$el.animate({ height: '0%' }, 100, function() {
+        // Change map to full size first so there isn't empty space when
+        // results window animates out
```

Comment:
This makes sense. The tricky part is Leaflet wants you to call `map.invalidateSize` when the map size has changed. This is currently being done after `.half` is removed from the map container: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/views.js#L220.

We may need to call it here in the animation callback directly or through another method on `App.map`